### PR TITLE
Select & Combobox - Make sure the focus state is the same as Orbiter's focus state

### PR DESCRIPTION
**Pull request by fraincs**
Fix for issue #593 : Select & Combobox - Make sure the focus state is the same as Orbiter's focus state

### CODE REVIEW

File: packages/Combobox/src/combobox.scss [hunk 1]
```diff
@@ -131,6 +131,11 @@
     &--active,
     &.active {
         border-color: var(--ids-combobox-border-color-active);
+
+        &:focus,
```
Comment (vicky-comeau) [inline packages/Combobox/src/combobox.scss:135]: I don't think this is supposed to be inside --active class

File: packages/Select/src/select.scss [hunk 2]
```diff
@@ -122,6 +122,11 @@
     &--active,
     &.active {
         border-color: var(--ids-select-border-color-active);
+
+        &:focus,
```
Comment (vicky-comeau) [inline packages/Select/src/select.scss:126]: same as above

File: packages/Combobox/src/combobox.scss [hunk 3]
```diff
@@ -131,6 +131,11 @@
     &--active,
     &.active {
```
Comment (vicky-comeau) [inline packages/Combobox/src/combobox.scss:132]: I think we need to specify a white background color for active. If not, we end up with a grey background when the menu is open:
image
This is taken for the select, but it's the same for the combobox.

File: packages/Combobox/src/combobox.scss (hunk 1 above)
Comment (fraincs) [reply packages/Combobox/src/combobox.scss:135]: Opening a Select/Combobox gives the select both the active and focus class. This is why this is inside active.

File: packages/Select/src/select.scss [hunk 4]
```diff
@@ -106,22 +113,27 @@
...
     }
 
     &--active,
     &.active {
+        background-color: var(--ids-select-background-active);
         border-color: var(--ids-select-border-color-active);
+
+        &:focus-visible,
+        &.focus {
+            background-color: var(--ids-select-background-active);
```
Comment (vicky-comeau) [inline packages/Select/src/select.scss:134]: is it showing the active border color when it has focus?

Comment (vicky-comeau) [reply packages/Select/src/select.scss:134]: doesn't look like it in the preview, I'd add the active border color here too. (Combobox too)

**Review by vicky-comeau — APPROVED** [review]

**Review by franckgaudin — APPROVED** [review]

**Pull request merged**